client/my-sites/stats/site.jsx

```diff
@@ -211,12 +212,14 @@ class StatsSite extends Component {
 			supportsPlanUsage,
 			supportsEmailStats,
 			supportsUTMStatsFeature,
+			supportsDevicesStatsFeature,
 		} = this.props;
 
 		let defaultPeriod = PAST_SEVEN_DAYS;
 
 		const shouldShowUpsells = isOdysseyStats && ! isAtomic;
 		const supportsUTMStats = supportsUTMStatsFeature || isInternal;
+		const supportsDevicesStats = supportsDevicesStatsFeature;
```

Comment:
I'm hiding it from internal blogs for now because it doesn't use real data yet. To see the module you need to apply the feature flag.